### PR TITLE
Changed Expert Only heading to Indie Hosts Only

### PR DESCRIPTION
**Pull request by anmoljagetia**
Fixes Issue #165.

### CODE REVIEW

**Comment by smcgregor** [comment]
You changed the html file instead of the subtemplate. Look for
new.html.subtemplate in the help app. When you open the pull request,
change the subtemplate but don't change the html file. This makes it easier
to merge, but you should run the build locally before pushing it.

**Comment by anmoljagetia** [comment]
Ah! I realise my mistake here, I didn't run the tests, I'll get back with the fix ASAP!

File: Help/new.html.subtemplate [hunk 1]
```diff
@@ -64,10 +64,10 @@
     <p>
       <strong>
         <span class="glyphicon glyphicon-eye-open"></span> 
-        Experts Only:
+        Indie Hosts Only:
       </strong> 
       Other applications built on Privly provide privacy only in special 
-      circumstances and should be viewed as "experts only."
+      circumstances and should be viewed as "Indie Hosts Only."
```
Comment (smcgregor) [inline Help/new.html.subtemplate]: How about for the description: "These applications provide privacy if you host your own content server. If you don't know what that means, you should not use these apps."

**Comment by smcgregor** [comment]
One comment above, also, please remove 3f25001 from the pull request.

**Comment by smcgregor** [comment]
Please leave a comment when you add commits. Otherwise I don't get an email notifying me to review the PR again.

File: Help/new.html [hunk 2]
```diff
@@ -326,4 +326,4 @@
    </div>
   </div>
  </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (smcgregor) [inline Help/new.html]: Please remove commits that change the HTML. You should only change the subtemplate.

**Comment by anmoljagetia** [comment]
Removed all changed to new.HTML, merged changed from the current master branch, all tests also pass :)

**Comment by smcgregor** [comment]
Too many commits for a small change. I'll merge after you either:

* Rebase the pull request from master and squash your commits into a single commit (hard/complicated)
* Open a new pull request containing a single commit and close this pull request (easy fast)

**Comment by smcgregor** [comment]
Closing in favor of https://github.com/privly/privly-applications/pull/224

**Pull request closed without merging**